tests/platform_tests/api/test_thermal.py

```diff
@@ -28,10 +28,26 @@
     pytest.mark.topology('any')
 ]
 
+@pytest.fixture(scope="class")
+def gather_facts(request, duthost):
+    # Get thermal truths from platform.json file
+    chassis_truth = duthost.facts.get("chassis")
```

Comment:
Since these values are part of `duthost.facts`, I suggest renaming this from `chassis_truth` to `chassis_facts`. I have done the same here: https://github.com/Azure/sonic-mgmt/pull/2209/files#diff-5d04a73b4b142807edb8c343db1d80dbR69

Please do the same for all "truth" variables to align naming.